crates/bevy_pbr/src/render/depth_prepass.rs

```diff
@@ -0,0 +1,786 @@
+use crate::{
+    draw_3d_graph, AlphaMode, MeshPipeline, MeshUniform, StandardMaterial, MeshViewBindGroup,
```

Comment:
I made `TransformBindGroup`, `MeshViewBindGroup` because the transform no longer exists. I'm not yet sure if that was the right succession.